web_src/less/_review.less

```diff
@@ -173,3 +173,15 @@ a.blob-excerpt:hover {
 .review-box > .segment {
   border: none !important;
 }
+
+.pull.files.diff [id] {
+  scroll-margin-top: 99px;
+
+  @media @mediaMd {
+    scroll-margin-top: 130px;
+  }
+
+  @media @mediaSm {
```

Comment:
You could add `@mediaMdAndDown` to `variables.less` and merge these blocks

Reply:
There already is a @mediaMdAndDown...